packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -70,6 +70,8 @@ Future<void> buildWeb(
         ? null
         : globals.flutterVersion.engineRevision,
       flutterRootDir: globals.fs.directory(Cache.flutterRoot),
+      // Web uses a different Dart plugin registry.
```

Comment:
Is there a tracking issue we could link to for moving the web plugin registry?

Reply:
Done. https://github.com/flutter/flutter/issues/80406